src/main/sh/mommy

```diff
@@ -181,7 +181,8 @@ fill_template() {
                           -e "s/%%THEY%%/$they/g" \
                           -e "s/%%THEM%%/$them/g" \
                           -e "s/%%THEIR%%/$their/g" \
-                          -e "s/%%CAREGIVER%%/$caregiver/g")"
+                          -e "s/%%CAREGIVER%%/$caregiver/g" \
+                          -e "s/%%N%%/\\$n/g")"
```

Comment:
this is the only format that i could find that works universally on all systems~